### ISSUE TITLE
PNI / IHR companion piece

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#3801]
IHR team is deprecating https://internethealthreport.org/ and also launching a companion piece for PNI. 

Same launch date as PNI - Nov 12.

### Blog category
- https://foundation.mozilla.org/blog/category/internet-health should be https://foundation.mozilla.org/blog/category/internet-health-report
- I can update this in the CMS but then it doesn't appear in the blog wagtail component 
- Kristina to file a dev ticket https://github.com/mozilla/foundation.mozilla.org/issues/3822

### Hub of IHR on Foundation site
- Kasia mocked this up a little [here](https://docs.google.com/presentation/d/19i7h2UqpEChTukeDxopMaHL_UJPA8zK21TmxSEuM6jY/edit#slide=id.p)
- a have created these pages here https://foundation.mozilla.org/en/internet-health-report/
- she will be updating the content
- once she is done we will need to give it a design review
- @gideonthomas is working with her on the Pulse Contributor portion of the site
- once this is all ready, devs will need to redirect https://internethealthreport.org/ to this new site and deprecate the old one

### PNI companion piece
- [their brief](https://docs.google.com/document/d/1rLtenQ2AUWZyS82h8SlOJpPYJ29UyJweWWtTTBfMEMI/edit?ts=5da9e9bc#heading=h.s3ksn9x63mux)
- Kasia has mocked this up a little [here](https://docs.google.com/presentation/d/19i7h2UqpEChTukeDxopMaHL_UJPA8zK21TmxSEuM6jY/edit#slide=id.p) but let's explore our options
- they would like some sort of landing page _(large house illustration, intro text, download button (pfd and other formats))_, thumbnail links to the 3 articles and a prominent link to PNI), there should also be an about page 
- 3 articles
  - [copy is here](https://docs.google.com/document/d/1N-PXJKNmek8vAj1FArePh--d2Hh42RSn4O7vHfqR1YI/edit#heading=h.n7cs1enai5cb)
  - wip example of a [printed article ](https://drive.google.com/drive/u/0/folders/1Kq8kvNQG7-TaZ7yTlVs0_PO53btf0jq6) _(site should draw from design elements used in the printed piece)_ (updated to single column: [211019-Securing-intro.pdf](https://github.com/mozilla/foundation.mozilla.org/files/3751642/211019-Securing-intro.pdf) [211019-single-column.pdf](https://github.com/mozilla/foundation.mozilla.org/files/3751643/211019-single-column.pdf) [211019-what-can-be-done.pdf](https://github.com/mozilla/foundation.mozilla.org/files/3751644/211019-what-can-be-done.pdf))
  - [wip illustrations](https://drive.google.com/drive/folders/1dOpK3KVXfoEUhSY9UltXkENtK0Vle7tn)
  - they would like to have comment feature for these articles
  - if we do these as a blog post page type, they would rather not show the date posted
- Note we have incredibly limited dev time for this because of PNI work, mozfest, mozfest travel time and in-lieu days, dev PTO, Canada's Remembrance day stat

**Design to explore options, initial ideas are:**

A) Make a regular landing page and then link out to 3 blog posts
- Pro
  - blog pages have share buttons
  - larger body text for easier reading
  - easier to add comment feature
- Con
  - harder for users to navigate between about pages and between different articles
  - probably can't hide the publish date

B) Group everything together
- Secondary nav would be _Overview, About, Article Title 1, Article Title 2, Article Title 3_
- Pro
  - easy for users to navigate and see all the info in one place
  - more wagtail component layout options
- Con
  - doesn't have share icons (but _might_ be able to add this)
  - might be weird to add comments

C) Another option?

cc @kaodro

**Comment by kristinashu** [comment]
@alanmoo we're trying to decide which templates to use for the PNI/IHR Companion articles. Either way, we will need to add a comment feature. I assume this is easy enough to do for blog posts but would it be possible to add it as a wagtail component that could be added to a regular wagtail page?

**Comment by kristinashu** [comment]
@beccaklam will test the two possible directions (make a regular landing page and then link out to 3 blog posts OR everything in a tabbed banner campaign page) to see which one feels best.

Alan, we would still need to know if we can add comments to a regular wagtail page (example, at the bottom of https://foundation.mozilla.org/en/internet-health-report/).

**Comment by kaodro** [comment]
I created a page and a blogpost with the same copy and image to compare (saved as drafts) and the blogpost definitely has more of the look we are looking for. It is mostly because of the narrow column - is this something we could customize on a page if we went for pages?

**Comment by beccaklam** [comment]
> I created a page and a blogpost with the same copy and image to compare (saved as drafts) and the blogpost definitely has more of the look we are looking for. It is mostly because of the narrow column - is this something we could customize on a page if we went for pages?

Unfortunately currently we would be unable to do that if we went with the tabbed page. If we want the blog layout we would need to link out to blog pages which we agree, has a better layout for the given content.

**Comment by kristinashu** [comment]
For non-blog pages, make sure you check "Narrowed page content" under the Settings tab, it will make the content less wide and much better!

<img width="746" alt="image" src="https://user-images.githubusercontent.com/1068783/67298931-dbbc3480-f4a0-11e9-9201-ce411737b040.png">

**Comment by beccaklam** [comment]
@kristinashu @kaodro Okay I played with some explorations and there are pros and cons to both. 

# **[Version 1](https://foundation.mozilla.org/en/initiatives/privacy-included/)**

### Pros
- Stories could be laid out using the blog style (larger text, share icons)
### Cons
- The user would need to click twice in order to get to a story, feels slightly more disjointed. I guess the payoff is a nicer reading experience

# [Version 2](https://foundation.mozilla.org/en/initiatives/privacy-included-3/)

### Pros
- User only needs to click once to get to stories
- Feels more cohesive/like 'one piece'
### Cons
- Can't lay out stories using blog style (smaller text, no share buttons)
- Fitting all the story titles on the horizontal nav might be tough

Let me know what you think!

**Comment by kristinashu** [comment]
One more version using regular "whitepaper style" opportunity pages...

# **[Version 3](https://foundation.mozilla.org/en/initiatives/privacy-included-v3/)**

### Pros
- Medium width (images are large but text length isn't too long)
- User only needs to click once to get to stories
- Feels more cohesive/like 'one piece'
- Vertical sub nav fits a little better than the horizontal one in V2

### Cons
- Can't lay out stories using blog style (smaller text, no share buttons)
- Fitting all the story titles on the horizontal nav feels a bit busy

**Comment by kristinashu** [comment]
But I'm leaning towards using the blog post style for the stories. I like how they feel more focused. We'll just need to have some intro/outro text to help frame each story in the context of the series.

Also might be nice if we could make the landing page just a single page (instead of having the credits on a separate page), then it could just be an opportunity instead of a bannered page.

**Comment by kaodro** [comment]
Thanks @kristinashu and @beccaklam !I wonder if we could combine Version 3 with the blogpost format for the stories? If this was all one page could the side menu items be jump links to a) stories as blogpost thumbnails b) Credits section?

**Comment by alanmoo** [comment]
Adding comments to the blog pages is something we’ve talked about doing and makes reasonable sense. IMO, it makes less sense to add comments to non-blog pages, but it could be done. In any case, it probably has to be baked into the template though, perhaps with a setting to turn them on/off for a page. I’m not sure how comments will play with localization though - we will likely have different comments for each language version of the page, and then someone needs to be responsible for moderation.

That all being said, I’m a bit concerned about timeline because this does require development effort that’ll span at least a week (not necessarily a week for a single dev, but spread between different responsibilities and availabilities) and we have zero dev capacity till November 4, and Ops’ focus is on the 2 donate projects through at least Nov 11.

**Comment by kristinashu** [comment]
Ok thanks of the input Alan. 

If comments are a must then let's go with the blog posts for the stories. @kaodro we won't be able to hid the publish date but I think it might actually be good to have the date, especially in the future so people can get a sense if it's out of date or not.

So the landing page could look like this [Version 1](https://foundation.mozilla.org/en/initiatives/privacy-included/) or a single page like this new [Version 4](https://foundation.mozilla.org/en/initiatives/privacy-included-v4/). 

I'll also ask Mavis if we can make the regular wagtail pages have a larger body text option like the blog posts pages (this is something we've thought about doing for a while now).

Talk soon!

And the stories would link out to [this blog example](https://foundation.mozilla.org/cms/pages/558/view_draft/) Rebecca set up (it's only a draft so it's trickier to view right now).

**Comment by kaodro** [comment]
Notes from our meeting today:
We'll go for [Version 1](https://foundation.mozilla.org/en/initiatives/privacy-included/) and use blogposts for the stories. It's ok to have the date and author (we'll use Internet Health Report). We will add the Category Internet Health Report and the tag "PrivacyIncluded". Under each story we want to show the other two stories as "related" and tag them with “Privacy included” so they never change.

To clarify:
1) How do people find the way back from the story/blogpost to home overview -> can we redirect an empty blogpost to overview page?
2)How can we do callouts in story 1 in a creative way?
3)Can we do GIfs (for the infographic)?

ToDo (designers, devs):
1) Add comments section to blogposts 
2) Mockup the infographic for story 2
3)Share landing page banner dimensions with Xenia
3) Once content is final (after Monday): mockup the final page + stories

ToDo editorial:
1) Add prompt for comments
2) Change copy for the landing page for online

**Comment by alanmoo** [comment]
It sounds like there are a few engineering asks here

• Some sort of GIF/animated graphic support (HTML based interactive animations would be better, but we're severely constrained on engineering availability) _NOTE: I'm cautious one because of the potential impact on SEO due to page size bloat. I’d want to restrict the file size or find another format that is better than GIF. And adding it might open a can of worms in terms of performance/SEO impact on the rest of the site_
• Tagging on opportunity pages so that we can get them to show up in the "related content" fields below blog posts
• Coral Talk comments on blog posts

Unfortunately, due to engineering constraints and other priorities, getting all of these done by the November 16 launch is not realistic. We might be able to get one done. @kaodro, how would you prioritize these 3 items?

**Comment by kristinashu** [comment]
I'll let Kasia confirm but the order is probably:
- comment functionality is the most important
- the *Privacy Included landing page showing up as a Related Post is less important because we can kinda do this manually by add them to the blog body section and just not use the Related Posts
- GIFs... yeah but have you considered the coolness factor?!?

**Comment by kaodro** [comment]
I confirm. So in the end there are just 2 tasks (comments and gifs) and yeah, the GIFs would be a big big coolness bonus.

**Comment by kristinashu** [comment]
@kaodro since we unfortunately might not have gif functionality, I don't think Xenia should bother creating them until we know for sure (unless you want them for social or something else like that).

@alanmoo one additional nice-to-have from devs would be the have the body text larger `.body-large` for the paragraph component so text is more legible at full width. In terms of priory I would put this after Comments (dev ticket for adding comments https://github.com/mozilla/foundation.mozilla.org/issues/3589).

Screen caps from site:
<img width="1150" alt="image" src="https://user-images.githubusercontent.com/1068783/67562275-40f76c00-f6d3-11e9-9437-06afabd76687.png">

<img width="1292" alt="image" src="https://user-images.githubusercontent.com/1068783/67562328-5a001d00-f6d3-11e9-9b8f-8b00fb0ea48c.png">

I've updated the landing page https://foundation.mozilla.org/en/initiatives/privacy-included/ and made it so anyone can see it with the password `privacyincluded` so Xenia, the illustrator, can see. I will also share [this example of a blog post](https://foundation.mozilla.org/en/blog/journey-dark-side/) with her in slack now as.

Also added image dimensions to her the [spreadsheet](https://docs.google.com/spreadsheets/d/1ARH-hAR-XPrXUz4ZbW1_7iQ7Relrkmteq6upAhjSpmM/edit#gid=0).

**Comment by kaodro** [comment]
Hi @kristinashu thank you!
Would it be possible to host this site under: https://foundation.mozilla.org/en/privacyincluded/? 
Next week we could start moving the content from the document to online. What's the best way to support you with the design of the blogpost/story pages?
The content and the illustrations (as well as the banner) should be final on Friday at latest.

**Comment by kristinashu** [comment]
> Would it be possible to host this site under: https://foundation.mozilla.org/en/privacyincluded/?

Yes, I'll set that up.

> Next week we could start moving the content from the document to online. What's the best way to support you with the design of the blogpost/story pages?

I can enter the copy and layout the pages for Privacy Included (landing page & 3 stories) and then you could just review and proof read them. 

@kaodro are you able to move the old blog posts over from https://internethealthreport.org/ and enter content to https://foundation.mozilla.org/en/internet-health-report/?

**Comment by kaodro** [comment]
Yes! I will do that next week. Thank you!

**Comment by kaodro** [comment]
@kristinashu The copy is now locked in the draft document. 
1) You can find the current PDF draft here: https://drive.google.com/drive/u/0/folders/1Kq8kvNQG7-TaZ7yTlVs0_PO53btf0jq6 (Lyall is now working on small improvements, see slack)
2) Pending decisions: Abigail will let me know if to blur the lioness image or not and if we need to credit the images.
3)When adding the stories please set the author as Internet Health Report and date as November 12, 2019 (I assume that's possible?)
4) Further reading should be under story 1 in the online version

Do you know when will I be able to review the first online version? I'm half-off on Thursday and would love to have it locked and ready by Wednesday, if possible. Thank you!

**Comment by kristinashu** [comment]
@kaodro here's where we're at with the pages. We will be making some small layout adjustments but I think they're ready for you to review and edit the copy.

pw: privacyincluded

https://foundation.mozilla.org/en/privacy-included/
- I think Solana mentioned adjusting the intro to better suite the online format
- will add the proper download links once they're finalized
- Story thumbnails aren't showing up property right now because they aren't published to the public yet, they will show up properly once they're published

Stories
https://foundation.mozilla.org/en/blog/how-smart-homes-could-be-wiser/
https://foundation.mozilla.org/en/blog/securing-internet-things/
https://foundation.mozilla.org/en/blog/5-key-decisions-every-smart-device/
- as we discussed, we should add some intro and outro copy to give these more context on their own

**Comment by kaodro** [comment]
Thank you! I will review the copy in detail tomorrow but wanted to ask about the following already now:

- Will these stories get their proper category "PrivacyIncluded"? The related stories on the landing page show our latest blogposts now.
- We were planning to add an empty post that would redirect people back to landing page after they read a story.
-Can we use big quotations marks when quoting Kathy in the first story?

Thank you!

**Comment by kristinashu** [comment]
@kaodro I've made a few layout updates to https://foundation.mozilla.org/en/internet-health-report/

For you Privacy Included questions:

> Will these stories get their proper category "PrivacyIncluded"? The related stories on the landing page show our latest blogposts now.

"PrivacyIncluded" will be a _tag_. I'm using the _category_ of "Internet Health" right now to show how it will look but once the stories have been published, I will update that component to only show posts with the tag "PrivacyIncluded". 

> We were planning to add an empty post that would redirect people back to landing page after they read a story.

No, we won't be able to do the redirect (Alan has explained that redirects only work if the page doesn't exist). We hope to be able to link to the landing page in the Related Posts using tagging but I don't think it will be ready for launch. This is also why I think we need to add the intro and outro text.

> Can we use big quotations marks when quoting Kathy in the first story?

Yes, updated!

**Comment by kaodro** [comment]
TY! They look really nice, thank you for the work.

Just a couple of little things:

1) For some reason the bold and cursive doesn't show up in editor's note in https://foundation.mozilla.org/en/blog/securing-internet-things/

2) The section under a chair: is there any way to make it look a bit cooler? Maybe more spacing between the bullet points?

3) The PI About section: I updated the list of names but couldn't figure out the spacing - could you correct it?

4) About the tag update: if it needs action from you I suggest we publish EOD for youo Nov 11 so that we have everything working for our Nov 12 in the morning.

**Comment by kristinashu** [comment]
> 1) For some reason the bold and cursive doesn't show up in editor's note in https://foundation.mozilla.org/en/blog/securing-internet-things/

Looks like italics works but for some reason bold isn't. This is a bug so I've opened a ticket for it https://github.com/mozilla/foundation.mozilla.org/issues/3847 but it probably won't get fixed in time for launch.

> 2) The section under a chair: is there any way to make it look a bit cooler? Maybe more spacing between the bullet points?

Updated.

3) The PI About section: I updated the list of names but couldn't figure out the spacing - could you correct it?

You just need to do a soft return (`shift` + `return`) instead of a hard return (just `return`).

4) About the tag update: if it needs action from you I suggest we publish EOD for you Nov 11 so that we have everything working for our Nov 12 in the morning.

Yes, sounds good!

5) Can we set the publication date on November 12?

You should be able to do that in the cms.

<img width="558" alt="image" src="https://user-images.githubusercontent.com/1068783/68236805-ba883780-ffba-11e9-8ae8-2e21f0b26596.png">

6) I am confused, is https://foundation.mozilla.org/en/privacy-included/about/ the same as https://foundation.mozilla.org/en/initiatives/privacy-included/about/ (or without about). Which one is the one we should work on?

Right, now that we have decided to use https://foundation.mozilla.org/privacy-included/ I've just deleted the one under /initiatives https://foundation.mozilla.org/en/initiatives/privacy-included/.

7) I just discovered I can't edit the https://foundation.mozilla.org/en/privacy-included/ version, can I get permissions @alanmoo ?

I'll ping Alan tomorrow.

**Comment by kristinashu** [comment]
@kaodro Alan says you should be able to edit those pages. But let us know if you're still having issues. Although you won't be able add new pages since it's directly under the home page and we try to keep these pages to a minimum.

**Comment by kaodro** [comment]
Thanks! No permissions so far but I already pinged him on Slack.

1) I added intros and outros to the articles. Solana added the rewritten About for Overview and About to the bottom of the [draft doc.](https://docs.google.com/document/d/1N-PXJKNmek8vAj1FArePh--d2Hh42RSn4O7vHfqR1YI/edit#) I can't add now cause of permission but will do it once I can or feel free to do it yourself!

2) I can set the publishing date on Monday. I discovered that once the date is set  it doesn't show any changes live so that was annoying.

3)When will we get our Internet Health Report category? FYI: When moving blogposts I am adding the i"nternet health" for now.

4) For Story1: we should maybe reuse the house at the very top of this one, and see if we can make the chair smaller so it’s clear that the section with the chair is more of a box. It feels clear for the product sections but not for the chair section yet. What do you think? I added the house on the top but didn't want to touch the chair to not mess it up.

5)Will we have comments for launch? :)

Thank you again!

**Comment by kristinashu** [comment]
> 1) I added intros and outros to the articles and the new About to the landing page. 

Great!

> 2) I can set the publishing date on Monday. I discovered that once the date is set  it doesn't show any changes live so that was annoying.

Ok, I can adjust it when I publish them on Nov 11. I'll put it as Nov 11 but we can always re-update it on Nov 12.

> 3) When will we get our Internet Health Report category? FYI: When moving blog posts I am adding the "internet health" for now.

Should be live today or tomorrow. I can update them then.

> 4) For Story1: we should maybe reuse the house at the very top of this one, and see if we can make the chair smaller so it’s clear that the section with the chair is more of a box. It feels clear for the product sections but not for the chair section yet. What do you think? I added the house on the top but didn't want to touch the chair to not mess it up.

I'll do a final design pass tomorrow (for this and all the pages). Please compile any design feedback and I'll look at it all at once tomorrow.

> 5) Will we have comments for launch? :)

Yes! Should go live today or tomorrow.

@kaodro who on your team should have access to moderate the comments?

**Comment by kaodro** [comment]
Thank you!

@kaodro who on your team should have access to moderate the comments?
-> Me, Solana, Jairus, Stefan.

Great! I'm not sure it needs "Also see: 5 Key Decisions for Every Smart Device | Securing the Internet of Things" since those links are just below in the Related Posts section.
-> I wondered the same but we concluded that sometimes people don't look at the related boxes and in this way we have it linked in a double way.

Design:
-> Thank you for all the awesome improvements! It looks really great now.
Question on bolding in the leading paragraph: It worked in Story1 (https://foundation.mozilla.org/en/blog/how-smart-homes-could-be-wiser/#what-can-be-done) but it doesn't work in Securing the IoT? (https://foundation.mozilla.org/en/blog/securing-internet-things/) Is it bc of the intro sentence in Story1?

-> I think I prefer not to do italic in the Chair section, what do you think? (the new design looks great btw).

-> I would also remove the italic in the Intro on the Landing page (https://foundation.mozilla.org/en/privacy-included/)

Lastly, just wanted to ping about the sign-up form in the https://foundation.mozilla.org/en/internet-health-report/ so we don't forget it. Solana is reviewing the copy now.

@solanasaurus If you have any more design feedback regarding the PI, add here please before Kristina wakes up.

**Comment by Solanasaurus** [comment]
Everything is looking great! So beautiful!

I have a little feedback.

Securing the Internet of Things: There’s needs to be a prominent mention of the DATE (April 2018) at the beginning of the text to make SURE people understand it’s not from now.

The author on ALL articles should be "Internet Health Report" instead of “Internet Health”.

And we need to make sure that the publication date for ALL articles is amended to NOVEMBER 13 so it matches the actual launch date and not October 23.

I can’t wait to see what the https://foundation.mozilla.org/en/privacy-included/ page looks like with the actual three articles that belong to the report listed!

The only page I feel like doesn’t have the right content, is the Overview page of the Internet Health Report. If this is the first thing you see when you click on the IHR starting from “Initiatives” I think it needs to be a bit more exciting than a typical “about page”. Maybe the latest 1-2 publications are featured? If this is our “front page” I’m not sure that’s where I would put our bios either.

Thank you so much!

**Comment by kristinashu** [comment]
Thanks @Solanasaurus and @kaodro! Few more notes:

I've very please to say the newsletter signup has been added to https://foundation.mozilla.org/en/internet-health-report/! Steph has added and confirmed it is working and functioning the same as on https://internethealthreport.org/.

> Great! I'm not sure it needs "Also see: 5 Key Decisions for Every Smart Device | Securing the Internet of Things" since those links are just below in the Related Posts section.
> I wondered the same but we concluded that sometimes people don't look at the related boxes and in this way we have it linked in a double way.

True! I've kept them there but added an image and the diving lines so it stands out more.

> Question on bolding in the leading paragraph: It worked in Story1 (https://foundation.mozilla.org/en/blog/how-smart-homes-could-be-wiser/#what-can-be-done) but it doesn't work in Securing the IoT? (https://foundation.mozilla.org/en/blog/securing-internet-things/) Is it bc of the intro sentence in Story1?

I think there's an issue where that style of text can't be both bold and italic. It is a bug but for now I think you can to choose just one at a time.

> I think I prefer not to do italic in the Chair section, what do you think? (the new design looks great btw).

Yeah, I've removed it.

> I would also remove the italic in the Intro on the Landing page (https://foundation.mozilla.org/en/privacy-included/)

Done!

I've also made a checklist to keep track of the outstanding things to do. They are mostly blocked by dev work but are close to being done and will be done in time for launch. Let me know if I've missed anything!

### Pre-launch checklist:
- [ ] Add Kasia, Solana, Jairus, and Stefan as blog moderators
- [ ] Update all stories from `Internet Health` to `Internet Health Report` category
- [ ] Add comment feature on all the stories
- [ ] Publish and remove password from all pages
- [ ] Update publish date to NOVEMBER 13 
- [ ] Update stories feed on https://foundation.mozilla.org/en/privacy-included/ to show stories with 'Privacy Included' tag (so it just shows the 3 stories)
- [ ] Update blog feed on https://foundation.mozilla.org/en/internet-health-report/blog/ to show `Internet Health Report` category instead of `Internet Health`

**Comment by Solanasaurus** [comment]
Thanks again Kristina!

A couple additional things for the checklist.

Make sure PDF has an appropriate filename, like Privacy_Included_IHR2019.pdf or whatever style we usually use (maybe more for us to remember)

The DOWNLOAD button on the main "Privacy Included: page also include a link to ePub and “other formats”. I guess we don’t have other formats?

I’ll finish editing the texts for the IHR pages as quickly as possible. Will try today!

Best,

Solana

**Comment by kaodro** [comment]
Thanks, both!
Solana is working on the copy for the IHR hub here: https://docs.google.com/document/d/1PLZdNG0aUdlQfvbqbz_d39_hQZcJhKWhxxNeERQ9liI/edit 
I hope to have the PDF and epub files today or Monday. We will name them appropriately. I think there will just be 2 types of files unless we have also a b&w easy print version (need to clarify with designers). 
I changed the author in all 3 articles to "Internet Health Report".
We are close! <3

**Comment by kaodro** [comment]
@kristinashu Will I have access to Google Analytics for our IHR hub and the publication?

**Comment by kristinashu** [comment]
Ok, I've updated the checklist! 

@kaodro are you the only one who needs access to Google Analytics?

**Comment by kaodro** [comment]
You can give access to all of us, just in case but I will be the one who
uses it for most of the time.  Thanks!

**Comment by kristinashu** [comment]
Alan has given all you access to Google Analytics for the foundation site!

@kaodro for Cade to add all of you as blog comment moderators, each of you will first need to login to comment component on the site https://foundation.mozilla.org/en/blog/securing-internet-things/. Make sure all of you use your foundation email. Once you've all done that please let me know!

**Comment by kaodro** [comment]
@kristina thank you! I registred with my Mofo account (I had a Talk account for our wordpress installation but I guess these are two separate accounts since the login didn't work). I asked the team to do the same and will ping you once they did.

**Comment by kaodro** [comment]
@kristinashu last day, wooo!
We will have 3 files today: a PDF, a black & white PDF and an epub. I will send it all to you via email and on Slack once I have them. 

Privacy_Included_IHR2019.pdf
Privacy_Included_bw_IHR2019.pdf
Privacy_Included_IHR2019.epub

Buttons:

Download full color PDF
Download black & white PDF
Download ePub 

Last question:
Is there a way to count PDF and Epub download in Google Analytics? If we need to set up an event, could someone on your team do it for us? 

Thank you again, it all looks soo good!

**Comment by Solanasaurus** [comment]
Hi from me too!

I updated the texts directly on the IHR pages using Wagtail and shifted some of the content around so the team list is on the contributor pages, and all texts are shorter. Feel free to tweak or beautify further. Thank you!

+1 on creating the event for tracking the download in Google Analytics. We don’t know how to do it and have forgotten in the past. 

Solana

**Comment by kaodro** [comment]
@kristinashu here are the files to upload: https://www.dropbox.com/sh/ll7vyc7d9uohc5s/AAAkjp9Z-ynOXe48SxdfQK6da?dl=0

You don't need to change the names, they are good as they are in dropbox!

**Comment by kristinashu** [comment]
I think we're all set!!! Here's the updated checklist.

### Pre-launch checklist:
- [x] Update all stories from `Internet Health` to `Internet Health Report` category
- [x] Add comment feature on all the stories)
- [x] Update blog feed on https://foundation.mozilla.org/en/internet-health-report/blog/ to show `Internet Health Report` category instead of `Internet Health`
- [x] Add Kasia, Solana, Jairus, and Stefan to Google Analytics
- [x] Add Kasia, Solana, Jairus, and Stefan as blog moderators - https://github.com/mozilla/foundation.mozilla.org/issues/3873
- [x] Add final PDF _(make sure PDF has an appropriate filename, like Privacy_Included_IHR2019.pdf)_
- [x] Add final ePub and “other formats” downloads
- [x] Finalize [copy](https://docs.google.com/document/d/1PLZdNG0aUdlQfvbqbz_d39_hQZcJhKWhxxNeERQ9liI/edit) on and layout for /internet-health-report
- [x] Publish and remove password from all pages
- [x] Once stories are published, make sure they are showing up on https://foundation.mozilla.org/en/privacy-included/
- [x] Check links on the foundation site to make sure they're going to https://foundation.mozilla.org/en/internet-health-report/ instead of https://internethealthreport.org/
- [ ] Update publish date to NOVEMBER 13 <- updated to Nov 12 but you can re-update it later
- [ ] Redirect https://internethealthreport.org/ to https://foundation.mozilla.org/en/internet-health-report/ - https://github.com/mozilla/foundation.mozilla.org/issues/3827 <- waiting for all blog posts to be transferred over

For analytics, we can't do it in google analytics because it's just a cms button so we can't add any code to it. @alanmoo has created this short link for the main PDF download button which I believe will track the number of clicks https://mzl.la/ihr-nov-2019.

**Comment by Solanasaurus** [comment]
Yay! Thank you Kristina.

November 12 is fine for publication date, you can tick that one off!

Let’s double check we are definitely tracking all downloads.

**Comment by Solanasaurus** [comment]
Let’s remember at some point to look for any links on MoCo homepage that may need to be changed too

**Comment by kristinashu** [comment]
Everyone should now be able to moderate blog posts comments by going to https://comments.mozillafoundation.org/. It shows all comments, included ones on PNI, so you'll have to look at the comment title to know which ones are for your blog posts. Unfortunately we can't integrate comment notifications with Slack. Cade says it's a big lift to do since Coral has now spun out to Vox Media and hosted our integration.

Alan has helped make a quick fix for the Blog page link. If you go to https://foundation.mozilla.org/en/internet-health-report/, "Our blog" will go directly to all your blog posts. While Pomax tries to find a fix, we can't use just the word "Blog" as a nav item. Should have it back to the way it was soon tho!

Finding all instances of https://internethealthreport.org/ will matter a little less once we can redirect that url to the new page. @kaodro just let me know once you've moved all the old blog posts over!

**Comment by kaodro** [comment]
@kristinashu @cadecairos all blogposts in all languages are transferred from internethealthreport.org to the foundation blog. Woohoo! You can archive the old site and redirect the url now. 

@kristinashu where can I see the number of PDF downloads? Should I look at the number of visit to the PDF url?

**Comment by kristinashu** [comment]
Ok, I've assigned the re-direct ticket to Cade now.

For the PDF download, it was at 29 hits earlier today. Unfortunately you're going to have to ping Alan once and a while for an update on the number since it sounds like our bitly login is complicated.

The blog tab is back and working now!

**Comment by Solanasaurus** [comment]
Are we tracking downloads of the other formats too or just the main PDF button? Checking in with Alan on numbers should be fine, as long as we have numbers :) 29! Woo!

Let’s make sure to thoroughly test the redirection links in all languages before we archive the site.

Solana

**Comment by Solanasaurus** [comment]
One more question: do you know if Privacy Included is discoverable in search engines yet? I have tried searching on the exact term in Google but not had any luck. Are there any settings to tweak on the backend so the pages are indexed?

**Comment by alanmoo** [comment]
I'm not sure what you're referencing when you say "downloads of the other formats". That number comes from the count of hits to the mz.la link.

SEO: Is Privacy Included linked to from other public pages, like the site homepage? That's how engines discover it.

**Comment by kristinashu** [comment]
> I'm not sure what you're referencing when you say "downloads of the other formats". That number comes from the count of hits to the mz.la link.

She means the ePub version and the black/white version. Which do not have bitly links so we are not currently tracking them.

<img width="400" alt="image" src="https://user-images.githubusercontent.com/1068783/68978746-7eba5280-07b0-11ea-8c3d-dd8e4ca812c5.png">

> SEO: Is Privacy Included linked to from other public pages, like the site homepage? That's how engines discover it.

It isn't currently featured on the homepage. It's mainly featured on https://foundation.mozilla.org/en/internet-health-report/ but that's also a new page. Jen is still really busy but we could ping her about adding it to the homepage.

**Comment by Solanasaurus** [comment]
Hi Kristina, can you remind me who was meant to do the changes on the PNI About pages to include the IHR companion text? I’m sorry to ask you again, I forgot what you replied last time, I just want to make sure it’s really there for tomorrow, because it’s not there yet :)

Solana

**Comment by kaodro** [comment]
@kristinashu @alanmoo We'd like to track all 3 types of downloads - is this about creating bitly links for all 3 of them and seeing the stats in bitly? I have access to bitly - I will create the remaining links and post them here for documentation. If there is anything else to do or something I am missing, let me know!

**Comment by alanmoo** [comment]
That's it!

**Comment by kristinashu** [comment]
@Solanasaurus the about page updates are on staging and should be pushed to production site later today.

Cade has done the re-directions and has made some notes about localized pages here https://github.com/mozilla/foundation.mozilla.org/issues/3827.

I think that's everything so gonna close this ticket 🎉

**Comment by kristinashu** [comment]
About pages are live https://foundation.mozilla.org/en/privacynotincluded/about/why!
